.pre-commit-config.yaml

```diff
@@ -175,3 +175,9 @@ repos:
         entry: python scripts/pandas_errors_documented.py
         language: python
         files: ^pandas/errors/__init__.py$
+    -   id: pg8000-not-installed-CI
```

Comment:
why do we need this?

Reply:
Just to prevent someone adding pg8000 in the `ci/deps` in the future which would prevent this test from running.

Reply:
i c ok sure